code/style.css

```diff
@@ -1,7 +1,140 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+/*This is the styling for the font*/
+body {
+  font-family:'Permanent Marker';
+  color: black;
+  background-color: aqua;
+}
+
+/*This is the styling for the header*/
+header { 
+  height: 100vh;
+  width: 100vw;
+  overflow: hidden;
+  position: relative;
+}
+
+video {
+width: 100%;
+height: auto;
+position: absolute;
+z-index: 1;
+}
+
+h1 {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  position: absolute;
+  text-align: center;
+  z-index: 2;
+  color: black;
+  font-size: 100px;
+  text-shadow: -2px -2px 3px black;
+}
+
+h2 {
+  text-align: center;
+  padding: 1rem;
+  margin: 1rem;
+  color: black;
+  font-size: 50px;
+}
+
+h3 {
+  position: top;
+  padding: 1rem;
+  margin: 1rem;
+  color: black;
+  font-size: 20px;
+  font-style: underline;
+}
+
+form {
```

Comment:
Maybe you can continue to add some /*comments in pseudo code*/ to divide your css in sections throughout the css file.